tools/shoestring/shoestring/lang/messages.pot

```diff
@@ -11,7 +11,7 @@ msgid "argument-help-announce-transaction-transaction"
 msgstr ""
```

Comment:
since you didn't add any messages, you should be able to revert lang changes (only changed line numbers in comments), but up to you

Reply:
yea can revert, wasn't sure if the line number change would cause an issue.